Identity/Tests/IdentityImplementingTests.swift

```diff
@@ -15,7 +15,9 @@ class IdentityImplementingTests: XCTestCase {
         let spy = MockAnalyticsLogger()
         let defaults = UserDefaults(suiteName: "test_identify_logged_userId_Persisted") ?? .standard
         let identityPersister = IdentityPersister(defaults: defaults)
-        let sut = IdentityFactory.makeModule(analyticsLogger: spy, identityPersister: identityPersister)
+        let sut = IdentityFactory.makeModule(analyticsLogger: spy,
+                                             identityPersister: identityPersister,
+                                             graphQLManager: MockGraphQLManager<Void>())
```

Comment:
Should we not test that the SSOProvider is set?

Reply:
do you mean checking if `sut.sso` not nil? I can add that test, but `sso` property is not optional. Do you think we still need to test it?

Reply:
OK, that's fine then